package.json

```diff
@@ -85,6 +85,6 @@
   },
   "dependencies": {
     "axios": "^0.19.2",
-    "uuid": "^3.4.0"
+    "uuid": "^8.3.0"
```

Comment:
I know this was previously reverted (#165), however I tried it out now in the web-app and it seems to work.

I'd say let's try it now and document what's wrong if we encounter an issue.